code/java/README.md

```diff
@@ -39,8 +39,8 @@ Add this dependency to your project's POM:
 
 ```xml
 <dependency>
-  <groupId>io.swagger</groupId>
-  <artifactId>swagger-java-client</artifactId>
+  <groupId>org.openapitools</groupId>
+  <artifactId>openapi-java-client</artifactId>
   <version>1.0.0</version>
```

Comment:
Hoort dit niet `<version>4.3.0</version>` te zijn? Volgens mij is [dit de release pagina](https://github.com/OpenAPITools/openapi-generator/releases/tag/v4.3.0) en anders haal ik dingen door elkaar.